QuizWhois.Api/Controllers/QuestionController.cs

```diff
@@ -23,22 +24,47 @@ public QuestionController(IQuestionService questionService, IQuestionRatingServi
         }
         
         [HttpPost]
-        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(CreatedResult), StatusCodes.Status201Created)]
         [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        public async Task<ActionResult> CreateQuestions(List<QuestionModel> questions)
+        public async Task<ActionResult> CreateQuestions(List<QuestionModelRequest> questions)
         {
-            await _questionService.CreateQuestions(questions);
-            return Ok();
+            var formedQuestions = await _questionService.CreateQuestions(questions);
+            if (formedQuestions.Count == 1)
```

Comment:
lets learn linq =) Single() is designed for this very case

Reply:
Single can throw exception if List have more, than 1 value, but i need to return result anyway

Reply:
correct, but SingleOrDefault doesn't throw. and yes, mb it is ok to use Count here